Hard/question_2.py

```diff
@@ -0,0 +1,19 @@
+"""
```

Comment:
**Choose different question from hard section**
Python doesn't have the "friend" concept explicitly, it provides a different mechanism for controlling access to class members based on naming conventions and conventions

Like, 
**Public members:** Members that are part of the class's public section and are intended to be accessed by users of the class are typically named without any leading underscores. For example, self.my_variable is a public member.

**Protected members:** Members intended for internal use within the class or its subclasses are typically named with a single leading underscore, like _my_variable. 

**Private members:** Python doesn't have true private members like C++ does. Instead, members that are intended to be kept as private and not accessed from outside the class are named with double leading underscores, such as __my_variable.